Mathlib/Tactic/CC/Datatypes.lean

```diff
@@ -130,7 +130,7 @@ structure CCConfig where
   /-- If `true`, then use excluded middle -/
   em : Bool := true
   /-- If `true`, we treat values as atomic symbols -/
-  values : Bool := true
+  values : Bool := false
```

Comment:
Can you explain here quickly why you decided to change this?

Reply:
This is a change in the dependent PR; I merge to master now...

Reply:
@joneugster Merged!